libraries/vault_installation_binary.rb

```diff
@@ -46,8 +46,14 @@ def action_create
           basename: options[:archive_basename])
 
         notifying_block do
-          directory ::File.join(options[:extract_to], new_resource.version) do
-            recursive true
+          [
```

Comment:
I wanted my change to be as non-breaking as possible, so I was concerned that if the full `options[:extract_to]` path didn't exist then it would create every parent directory. So if the option was set to `/some/deeply/nested/path/that/did/not/exist` the result would still be the same as the version of this cookbook before my change. 

I would be happy to remove the recursive attribute though, since that edge case does seem a bit excessive.